emma-language/src/main/resources/test_config.properties

```diff
@@ -0,0 +1,2 @@
+#See BaseCompilerSpec.compileSpecPipelines
```

Comment:
Why keep the `test_config.properties` in `src/main/resources` and `src/test/resources`?